packages/console/src/components/Pagination/index.module.scss

```diff
@@ -4,52 +4,75 @@
   display: flex;
   justify-content: flex-end;
   align-items: center;
-}
 
-.positionInfo {
-  font: var(--font-body-medium);
-  color: var(--color-text-secondary);
-}
+  .positionInfo {
+    font: var(--font-body-medium);
+    color: var(--color-text-secondary);
+  }
 
-.pagination {
-  display: flex;
-  justify-content: right;
-  margin: 0;
-  height: 28px;
-  padding-inline-start: _.unit(4);
+  .pagination {
+    display: flex;
+    justify-content: right;
+    margin: 0;
+    padding-inline-start: _.unit(4);
 
-  li {
-    list-style: none;
+    li {
+      list-style: none;
 
-    &:not(:first-child) {
-      margin-left: _.unit(2);
-    }
+      &:not(:first-child) {
+        margin-left: _.unit(2);
+      }
 
-    .button {
-      border-radius: 6px;
-      min-width: 28px;
-      padding: 0 6px;
-      height: 28px;
-      text-overflow: unset;
-      border: 1px solid var(--color-border);
+      &:hover {
+        background: var(--color-neutral-95);
+      }
 
-      > span {
-        margin: 0 auto;
+      &.disabled {
+        .button {
+          cursor: not-allowed;
+          background: var(--color-neutral-95);
+        }
       }
+    }
+  }
+
+  &.tiny {
+    .pagination {
+      height: 20px;
 
-      &.active {
-        border-color: var(--color-text-link);
-        color: var(--color-text-link);
+      li {
+        .button {
+          border-radius: 4px;
+          height: 20px;
+          border: unset;
+          background: unset;
+          padding: 0;
+        }
       }
     }
+  }
 
-    &.disabled {
-      .button {
-        cursor: not-allowed;
-        background: var(--color-neutral-95);
+  &.default {
```

Comment:
should use tiny to overwrite the styling? Is the default class name necessary?

Reply:
updated.